README.md

```diff
@@ -1,6 +1,127 @@
 Chitter Challenge
 =================
 
+# Using the program
+
+* Fork the repo
+* Run bundler
+
+## RSpec
+
+* run rspec to see coverage and tests
+
+## Rackup
+
+* run rackup to host the app on local ip address
+
+## My approach:
+
+### Step 1: Fully read user stories and create domain models
+
+- I created the domain, database and req/res models below from the first 4 user stories
+
+As a Maker  
+So that I can let people know what I am doing    
+I want to post a message (peep) to chitter
+
+As a maker  
+So that I can see what others are saying    
+I want to see all peeps in reverse chronological order
+
+As a Maker  
+So that I can better appreciate the context of a peep  
+I want to see the time at which it was made  
+
+As a Maker  
+So that I can post messages on Chitter as me  
+I want to sign up for Chitter  
+
+![Chitter Domain Model (1)](https://user-images.githubusercontent.com/74908625/112628934-891c4000-8e2b-11eb-97a0-3a284fa2790a.jpg)
+
+![Chitter Domain Model](https://user-images.githubusercontent.com/74908625/112629115-c7b1fa80-8e2b-11eb-9821-58bd0b6b86a7.jpg)
+
+<img width="382" alt="Screenshot 2021-03-29 at 09 10 19" src="https://user-images.githubusercontent.com/74908625/112806398-a3922b80-906e-11eb-8872-3780fa043738.png">
+
+### Step 2: Implement User story 1, 2 & 3
```

Comment:
I enjoy how you've walked through your entire modelling process and then outlined your approach - makes it very easy to follow